docs/off-topic/recording-videos.md

```diff
@@ -0,0 +1,25 @@
+**Table of contents**
+
+<!-- TOC depthFrom:1 insertAnchor:false orderedList:true updateOnSave:true -->
+
+- [Video Recording](#video-recording)
+- [Audio recording](#audio-recording)
+- [Video assembling](#video-assembling)
+
+<!-- /TOC -->
+
+# Video Recording
+
+- We use OBS for recording one screen in maximum size at 24 fps and export as MP4 file, recording one file per item to show, in order to reduce the amount of re-recording needed if something goes wrong. We don't care about audio here as we'll be recording audio at a later point
+
+# Audio recording
+
+- Audacity is used to record the audio snippets while watching the videos recorded in prior step, so that audio lenght is more or less similar to video.
```

Comment:
```suggestion
- Audacity is used to record the audio snippets while watching the videos recorded in the prior step, so that audio length is more or less similar to video.
```